### PR TITLE
Add missing specification examples in tests

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: vxquery-xtest/src/test/resources/Queries/XQuery/Json/Array/Navigation/q01_array_navigation.xq [hunk 1]
```diff
@@ -17,5 +17,5 @@
 
 (: Json Array Navigation Query :)
 (: Just navigate through an array :)
-    let $x:=[2,5]
-    return $x(1)
+let $wd := ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+return $wd(1)
```
Comment (westmann) [inline vxquery-xtest/src/test/resources/Queries/XQuery/Json/Array/Navigation/q01_array_navigation.xq]: Add a new line?

File: vxquery-xtest/src/test/resources/Queries/XQuery/Json/Array/Navigation/q02_array_navigation.xq [hunk 2]
```diff
@@ -17,6 +17,9 @@
 
 (: Json Array Navigation Query :)
 (: Just navigate through an array :)
-    let $x:=[1,[2]]
-    return $x(2)(1)
+let $x := [
+    [ "mercury", "venus", "earth", "mars" ],
+    [ "monday", "tuesday", "wednesday", "thursday" ]
+]
+return $x(2)(2)
```
Comment (westmann) [inline vxquery-xtest/src/test/resources/Queries/XQuery/Json/Array/Navigation/q02_array_navigation.xq:24]: Remove the white space before the new line at the end of the file?

**Comment by westmann** [comment]
Looks good! Just 2 whitespace fixes before we commit.

**Comment by riyafa** [comment]
Thanks. I have updated PR based on the suggestions

**Pull request merged**